barcodes_generator_product/README.rst

```diff
@@ -0,0 +1,134 @@
+==============================
+Generate Barcodes for Products
+==============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--barcode-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-barcode/tree/13.0/barcodes_generator_product
+    :alt: OCA/stock-logistics-barcode
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/stock-logistics-barcode-13-0/stock-logistics-barcode-13-0-barcodes_generator_product
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/150/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module expands Odoo functionality, allowing user to generate barcode
+depending on a given barcode rule for Products.
+
+For example, a typical pattern for products is  "20.....{NNNDD}" that means
+that:
+* the EAN13 code will begin by '20'
+* followed by 5 digits (named Barcode Base in this module)
+* and after 5 others digits to define the variable price
+* a 13 digit control
+
+With this module, it is possible to:
+
+* Assign a pattern (barcode.rule) to a product.product
+
+* Define a Barcode base:
+    * manually, if the base of the barcode must be set by a user. (typically an
+      internal code defined in your company)
+    * automaticaly by a sequence, if you want to let Odoo to increment a
+      sequence. (typical case of a customer number incrementation)
+
+* Generate a barcode, based on the defined pattern and the barcode base
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, see the 'Configuration' Section of the description
+of the module 'barcodes_generator_abstract'
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to a Product form (or a template form):
+
+1 for manual generation
+    * Set a Barcode Rule
+    * Set a Barcode Base
+    * click on the button 'Generate Barcode (Using Barcode Rule)'
+
+.. image:: https://raw.githubusercontent.com/barcodes_generator/static/description/product_template_manual_generation.png
```

Comment:
This link is not being generated correctly, but maybe that is a bug in `oca-gen-addon-readme`. The correct one should be:
```suggestion
.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/13.0/barcodes_generator_product/static/description/product_template_manual_generation.png
```

Reply:
Yes sorry, it was bad defined on USAGE.rst solved

Reply:
You must use the relative link in fragments (`../static/src/<etc>`), and let the script to generate the absolute one.